MapboxNavigation-Documentation.podspec

```diff
@@ -43,9 +43,9 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.module_name = "MapboxNavigation"
 
-  s.dependency "MapboxDirections.swift", "~> 0.9.0"
-  s.dependency "Mapbox-iOS-SDK", "~> 3.5"
-  s.dependency "OSRMTextInstructions", "~> 0.1.2"
+  s.dependency "MapboxDirections.swift", "~> 0.10.0"
+  s.dependency "Mapbox-iOS-SDK", "~> 3.6"
```

Comment:
If I’m not mistaken, this library is still compatible with v3.5, so leaving this line alone will allow developers to upgrade from v3.5.x to v3.6.0 at their own pace.